src/main/java/org/noureddine/joularjx/monitor/MonitoringHandler.java

```diff
@@ -247,47 +258,91 @@ private Map<Thread, Map<CallTree, Integer>> extractCallTreesStats(Map<Thread, Li
         return stats;
     }
 
+//    /**
+//     * Updates the CPU times for each Thread.

```

Comment:
I'd rather not comment  existing methods, but modify them (or remove), as it's easier to see the changes with your commit